e2e-tests/development-runtime/cypress/integration/navigation/linking.js

```diff
@@ -110,7 +110,7 @@ describe(`navigation`, () => {
 
   describe(`Supports unicode characters in urls`, () => {
     it(`Can navigate directly`, () => {
-      cy.visit(`/안녕`).waitForRouteChange()
+      cy.visit(encodeURI(`/안녕`)).waitForRouteChange()
```

Comment:
Chrome actually make encoded request - it's just ui nicety that it keeps unicode characters - this is needed for Cypress 3.5.0+ to work - otherwise we get "Could not load page" error from cypress (not even from our runtime / server)

![Screenshot 2020-12-11 at 02 46 16](https://user-images.githubusercontent.com/419821/101852835-cd2a5400-3b5e-11eb-851e-64c39c6c6828.png)